src/components/GGanttLegend/GGanttLegend.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="g-gantt-legend">
+    <g-gantt-legend-non-collapsed-view
+      v-if="!isCollapsed"
+      v-model:is-collapsed="isCollapsed"
+      :arrow-name="arrowName"
+      :legend="legend"
+    />
+    <g-gantt-legend-collapsed-view
+      v-else
+      v-model:is-collapsed="isCollapsed"
+      :arrow-name="arrowName"
+      :legend="legend"
+    />
+  </div>
+</template>
+<script setup lang="ts">
+/// <reference lib="es2017.object" />
```

Comment:
?

Reply:
I included types to Object.entries() to be working.
It should be in "GGanttLegendCollapsedView" (I moved it already).